cmd/fleetctl/package.go

```diff
@@ -74,11 +74,6 @@ func packageCommand() *cli.Command {
 				Value:       "com.fleetdm.orbit",
 				Destination: &opt.Identifier,
 			},
-			&cli.StringFlag{
-				Name:        "version",
-				Usage:       "Version for package product",
-				Destination: &opt.Version,
```

Comment:
Can we remove `Version` member?

Reply:
There are a bunch more file/test changes that are needed to remove it. I didn't want to risk breaking something.